core/arch/arm/pta/system.c

```diff
@@ -11,6 +12,17 @@
 
 #define MAX_ENTROPY_IN			32u
 
+/*
```

Comment:
Shouldn't these defines go into `pta_system.h` since it's part of the API?

Reply:
Sure, will fix.